subsys/bluetooth/host/hci_core.c

```diff
@@ -5672,8 +5677,22 @@ struct net_buf *bt_buf_get_cmd_complete(s32_t timeout)
 	return bt_buf_get_rx(BT_BUF_EVT, timeout);
 }
 
-struct net_buf *bt_buf_get_evt(u8_t evt, s32_t timeout)
+struct net_buf *bt_buf_get_evt(u8_t evt, bool discardable, s32_t timeout)
 {
+#if defined(CONFIG_BT_DISCARDABLE_POOL)
+	if (discardable) {
+		struct net_buf *buf;
+
+		buf = net_buf_alloc(&discardable_pool, timeout);
+		if (buf) {
+			net_buf_reserve(buf, CONFIG_BT_HCI_RESERVE);
+			bt_buf_set_type(buf, BT_BUF_EVT);
+		}
+
```

Comment:
I guess we should log something if we cannot allocate a buffer and thus would discard the event.

Reply:
The HCI drivers (at least the H4 one) have the logs for when they discard events.

Reply:
Depending on the LOG domain that may not be printed though and since this is behind a config option it may be difficult to analize traces so I don't think it would hurt to say we are dropping here, it doesn't need to be an error/warning though since since it been flagged as discardable.

Reply:
Sure, I'll add a log, it's not quite dropping that's happening here, rather just a failed allocation. It's then up to the driver to deal with this failure, either by dropping the incoming packet or by deferring to wait in a thread.

Reply:
Actually, can we forget about this for now. I'm trying to keep the changes minimal since this is to be backported, and adding such a log would mean that we should add it to `bt_buf_get_cmd_complete()` and `bt_buf_get_rx()` as well for consistency. In fact, the place you're asking to add it is the least important since discardable buffers are designed to be able to get allocation failures where as for the other pools it's a more severe thing.

Reply:
Sure, np.

Reply:
> Sure, np.

@Vudentz any chance you could withdraw your "changes requested' then? :)